bluebrain/repo-bluebrain/packages/py-bluepy/package.py

```diff
@@ -26,7 +26,6 @@ class PyBluepy(PythonPackage):
     depends_on("py-morph-tool@2.4.3:2", type=("build", "run"))
     depends_on("py-morphio@3.0.1:3", type=("build", "run"))
     depends_on("py-voxcell@3", type=("build", "run"))
-    depends_on("py-bluepysnap@0.13.0:0", type=("build", "run"), when="@:2.4")
```

Comment:
We should also remove bluepy 2.4.5 above (it's probably unused, but it would be broken without snap)

Reply:
ok, done